pkg/apis/serving/v1beta1/inference_service_validation.go

```diff
@@ -141,6 +146,40 @@ func validateInferenceService(isvc *InferenceService) (admission.Warnings, error
 	return allWarnings, nil
 }
 
+// Validate multi-node variables(tensor-parallel-size, pipeline-parallel-size, WorkerSpec.Size)
+func validateMultiNodeVariables(isvc *InferenceService) error {
+	if isvc.Spec.Predictor.Model != nil {
+		if envPipelineParallelSize, exists := utils.GetEnvVarValue(isvc.Spec.Predictor.Model.PredictorExtensionSpec.Container.Env, constants.PipelineParallelSizeEnvName); exists {
+			// pipelineParallelSize should be bigger than 1 (head + worker)
```

Comment:
In the error message, can we also mention that this should be head + worker

Reply:
Updated.